src/index.js

```diff
@@ -104,6 +107,9 @@ export function run(
   }
 
   // Handle sync call by default
+  if (typeof options.input !== 'undefined') {
+    nextOptions.input = options.input
+  }
```

Comment:
This change doesn't seems to be as part of this PR. Same in README. Let's keep PR atomic to the change which is the main subject